_includes/codepens/advtable-plugin/index.html

```diff
@@ -0,0 +1,160 @@
+<textarea id="advtable-plugin">
```

Comment:
The directory of this codepen and the id being used is a bit inconsistent, as all the others just use the name rather than adding the "-plugin" suffix. Not sure if you want to fix this, but just wanted to point it out.

Reply:
I wonder if we should move all plugins to a subdirectory, as that might be cleaner than using suffixes?

Reply:
updated

Reply:
we can move them next year